src/hooks/useTopicIcon.js

```diff
@@ -0,0 +1,12 @@
+import { useSelector } from 'react-redux';
+import _ from 'lodash';
```

Comment:
same comments as the previus PR with lodash, you can do
```suggestion
import find from 'lodash/find';
```